azure-go/go.mod

```diff
@@ -3,6 +3,7 @@ module ${PROJECT}
 go 1.23
 
 require (
-	github.com/pulumi/pulumi-azure-native/sdk/v3 v3.8.0
-	github.com/pulumi/pulumi/sdk/v3 v3.198.0
+	github.com/pulumi/pulumi-azure-native-sdk/resources/v3 v3.8.0
+	github.com/pulumi/pulumi-azure-native-sdk/storage/v3 v3.8.0
+	github.com/pulumi/pulumi/sdk/v3 v3.199.0
```

Comment:
This is the key change here, that restores what we had in before https://github.com/pulumi/templates/pull/961/files#diff-31c6663b34f8beca4dd7b7fdb13f45a96b6417b4ee9fad3e07792e217f9fd10cL6-L8, except with a higher version number.